variables.tf

```
@@ -49,6 +55,18 @@ variable "tags" {
   type        = map(string)
 }
 
+variable "use_keypair" {
+  default     = false
+  description = "Whether or not to associate an SSH Keypair with this instance. If this is false and no external_keypair is defined, no key will be associated with the instance."
+  type        = bool
+}
+
+variable "use_ssm" {
+  default     = false
```

IMO the default for this should be `true`

```suggestion
  default     = true
```